index.html

```diff
@@ -53,95 +56,108 @@ <h1 class="first-introduction">I'm Raphael.<br /> I am so glad to see you!</h1>
         <!-- Start of Project 1 -->
         <div id="tonic-project">
             <div class="data">
-                <img src="./images/snapshots/Snapshoot Portfolio.png" alt="Tonic project snapshot">
-                <h2 class="project-title">Tonic</h2>
-                <div class="work-nature">
-                    <label class="canopy">CANOPY</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="backend"> Back End Dev</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="year">2015</label>
+                <div class="image-div"><img src="./images/snapshots/Snapshoot Portfolio.png" alt="Tonic project snapshot"></div>
+                <div class="desktop-image-div"><img class="desktop-snapshop" src="./images/desktop/Snapshoot-Desktop1.png" alt="Tonic project snapshot"></div>
+                <div class="project-content-1">
+                    <h2 class="project-title">Tonic</h2>
+                    <div class="work-nature">
+                        <label class="canopy">CANOPY</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="backend"> Back End Dev</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="year">2015</label>
+                    </div>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="tools">
+                        <li><label class="tool-press">HTML</label></li>
+                        <li><label class="tool-press">CSS</label></li>
+                        <li><label class="tool-press">JAVASCRIPT</label></li>
+                    </ul>
+                    <button class="see-project">See Project</button>
                 </div>
-                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="tools">
-                    <li><label class="tool-press">HTML</label></li>
-                    <li><label class="tool-press">CSS</label></li>
-                    <li><label class="tool-press">JAVASCRIPT</label></li>
-                </ul>
-                <button class="see-project">See Project</button>
             </div>
         </div>
         <!-- End of Project 1 -->
 
         <!-- Start of Project 2 -->
         <div id="tonic-project">
```

Comment:
- [ ] The margins between your work cards are much compared to the template. Kindly fix this, please.
### Your design
![image](https://user-images.githubusercontent.com/94551484/229280726-c8f7a0e8-03b2-4255-9ad6-24b253858027.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/229280750-ccdb2718-5d66-4b2d-bd69-c1ea136b1a10.png)